components/raftstore/src/store/fsm/apply.rs

```diff
@@ -629,6 +630,16 @@ where
             }
             self.kv_wb_last_bytes = 0;
             self.kv_wb_last_keys = 0;
+        } else {
+            fail_point!(
+                "after_write_to_db_skip_write_node_1",
+                self.store_id == 1,
+                |_| { unreachable!() }
+            );
+            // We call `clear` here because some WriteBatch impl may have some interal state
+            // that need to be reset even if the write batch is empty.
+            // Please refer to `RegionCacheWriteBatch::clear` for more details.
```

Comment:
```suggestion
            // We call `clear` here because some WriteBatch impl may have some
            // internal state that need to be reset even if the write batch is empty.
            // Please refer to `RegionCacheWriteBatch::clear` for more details.
```